awx/main/utils/plugins.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2015 Ansible, Inc.
```

Comment:
FYI, the copyright year is for when content got created. Since you just wrote this code, it's probably
```suggestion
# Copyright (c) 2024 Ansible, Inc.
```
in this case.